### ISSUE TITLE
VDI Stream Host

### ISSUE THREAD

**Issue by avently** [mbroemme/vdi-stream-client#6]
Hello.
Parsec doesn't support Linux hosts but for me it's must have feature. I don't use Windows. I would like to have a good arm64/amd64 alternative to Anydesk for remote work via cloud servers. Seems like your app can be that alternative.
In readme you wrote:

> VDI Stream Host application seems reasonable

Do you have plans for creating the host application in the future that supports Linux?

**Comment by mbroemme** [comment]
Yes, actually it can be made with help of FFmpeg and couple of other open source technologies running on Windows. Also on the other side, there might be a chance that Parsec will release a version with Linux host support.

**Comment by avently** [comment]
Comparison table in Readme clearly says why your solution is better than alternatives. So even if Parsec will make a Linux support there is still a room for VDI Stream Host.

**Comment by mbroemme** [comment]
Yes, this is ideal solution as it will be completely open source and we can add enhanced features like raw clipboard data sharing instead of text only. We are not there yet and I want to focus on stabilization currently and make client code more portable.

**Comment by avently** [comment]
@mbroemme glad to hear that you still interested in adding new features and enhancing your app

**Comment by mbroemme** [comment]
It's in progress.
